src/database/upgrade/9/monitoring_sqlserver.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE #ofTmpStatus
+(
+    roomID        BIGINT IDENTITY(1,1) PRIMARY KEY,
+    roomJID       VARCHAR(1024) NOT NULL,
+    roomDestroyed TINYINT DEFAULT 0
+);
+
+INSERT INTO #ofTmpStatus (roomJID)
+SELECT DISTINCT room
+FROM ofConversation;
+
+UPDATE #ofTmpStatus
+SET roomDestroyed = 1
+WHERE roomJID NOT IN (SELECT ofMucRoom.name + '@' + ofMucService.subdomain + '.' + ofProperty.propValue
+                      FROM ofMucRoom
+                               JOIN ofMucService ON ofMucRoom.serviceID = ofMucService.serviceID
+                               CROSS JOIN ofProperty
+                      WHERE ofProperty.name = 'xmpp.domain');
+
+CREATE TABLE ofMucRoomStatus
+(
+    roomID        BIGINT        PRIMARY KEY,
+    roomJID       VARCHAR(1024) NOT NULL,
+    roomDestroyed TINYINT DEFAULT 0
+);
+
+INSERT INTO ofMucRoomStatus (SELECT * FROM #ofTmpStatus);
```

Comment:
Syntax checker does not like this INSERT query, and fails on the `*` of the sub-select.